Dungeoneer/src/com/interrupt/dungeoneer/entities/Monster.java

```diff
@@ -562,8 +576,14 @@ else if(alerted && playerdist <= 1f && !fleeing) // If close enough to the playe
 			targety = player.y;
 			stuckWanderTimer = 0f;
 		}
+
+		// Wander when not alerted
+		boolean canWander = wanders && !alerted && ambushMode != AmbushMode.None;
+
+		// Wander when there is no other path to the target
+		boolean stuckWander = !canSafelyNavigateToTarget || !chasetarget || stuckWanderTimer > 0;
```

Comment:
I appreciate this. 👍

Reply:
Yes, makes that part much more readable.